locales/en/app.ftl

```diff
@@ -888,26 +896,55 @@ site-nav-ad-callout = Try our other security tools:
 brand-relay = { -brand-relay }
 brand-mozilla-vpn = { -brand-mozilla-vpn }
 
+mobile-menu-label = Main menu
+
+main-nav-button-collapse-label = Collapse menu
+main-nav-button-collapse-tooltip = Collapse menu
+main-nav-button-expand-label = Expand menu
+main-nav-button-expand-tooltip = Expand menu
+main-nav-label = Navigation
+main-nav-link-home-label = Home
+main-nav-link-dashboard-label = Dashboard
+main-nav-link-settings-label = Settings
+main-nav-link-faq-label = FAQs
+main-nav-link-faq-tooltip = Frequently asked questions
+
 ## User menu
 
+# Deprecated
```

Comment:
nit: obsolete is probably more accurate than deprecated.

Reply:
Ah, I've been using `# Deprecated` for all strings I'm intending to remove so far; should I switch to using Obsolete in the future, or is that just for a specific situation in which we're planning to remove strings?

Changed all instances for now: 53ef0aee9